lib/registries/addon/components/registries-services-list/styles.scss

```diff
@@ -31,11 +31,27 @@
     background-repeat: no-repeat;
     background-position: center;
 
+    &.provider-asist {
+        background-image: url('img/provider_logos/ASIST_logo.png');
```

Comment:
I guess this doesn't hurt anything, but why not just put img tags in the template?

Reply:
I just followed the pattern that was already there, but using `<img>` is better here

Reply:
Ah, gotcha.